lib/opal/sprockets/server.rb

```diff
@@ -4,7 +4,14 @@
 require 'rack/builder'
 require 'rack/deflater'
 require 'rack/directory'
-require 'rack/showexceptions'
+
+# rack changed some paths in 2.0.0alpha
+begin
+  require 'rack/showexceptions'
+rescue LoadError
+  require 'rack/show_exceptions'
+end
+
```

Comment:
rack defines `RELEASE` constant inside 'rack.rb', so it could be accessed only if full rack would be required